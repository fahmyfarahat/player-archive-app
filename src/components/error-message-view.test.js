import React from "react";
import { shallow } from "enzyme";
import ErrorMessageView from "./error-message-view";
import toJson from "enzyme-to-json";

describe("ErrorMessageView - general", () => {
  let wrapper;
  beforeEach(()=> {
    const props = {
      message: "Player is not available.",
      className: "error"
    }
    wrapper = shallow(<ErrorMessageView {...props}/>);
  });
  
  it("should render without crashing", () => {
    expect(wrapper).toHaveLength(1);
    expect(wrapper.find(".error").text()).toEqual("Player is not available.");
  });

  it("should match Snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
