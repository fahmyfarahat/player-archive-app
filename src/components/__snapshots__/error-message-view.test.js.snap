// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessageView - general should match Snapshot 1`] = `
<SectionWrapper>
  <h1
    className="error"
  >
    Player is not available.
  </h1>
</SectionWrapper>
`;
